src/pages/components/testimonial/TestimonialCard.js

```diff
@@ -1,21 +1,25 @@
 import React from 'react';
+import { LazyLoadImage } from 'react-lazy-load-image-component';
+import { Link } from 'react-router-dom';
 
 const TestimonialCard = ({ item }) => {
   return (
     <figure className='relative pr-5 pb-5'>
-      {/* Image */}
-      <img
-        className='w-full rounded-lg object-cover drop-shadow-md'
-        src={item.src}
-        alt={item.name}
-      />
-      {/* Content */}
-      <figcaption className='absolute left-0 right-0 bottom-5 mr-5 rounded-lg bg-gradient-to-t from-black pb-5 text-white'>
-        <p className='mx-4 mb-4 text-sm font-semibold xl:text-lg'>
-          {item.text}
-        </p>
-        <p className='mx-4 font-allison text-[32px]'>{item.author}</p>
-      </figcaption>
+      <Link to={`/swipet-journeys/${item.author}`} target='none'>
+        {/* Image */}
+        <LazyLoadImage
```

Comment:
```suggestion
        <img
```

pwede ra ni <img tag, kay sa settings niya nag gamit naman ug lazyload
![image](https://user-images.githubusercontent.com/79126024/168790037-61c5688d-df0b-4521-b9bf-f2e8f32a01a4.png)